pkg/lib/manifest2po.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
```

Comment:
Probably at least a minimal licence header is worth having here (even if the old one didn't have it).  I'd be happy with GPL3+.

Reply:
uh ups, sorry; set LGPL 2.1+ as used (at least to me) everywhere